task9.py

```diff
@@ -0,0 +1,42 @@
+from datetime import *
+import portion as p
+from collections import *
+'''
+Take two datetimes from user start_time and end_time. Calculates the difference between start_time and end_time by 
+removing night intervals (12 AM to 6 AM). (Hint: You can use python intervals module)
+'''
+
+startdate_str = datetime(2023,1,10,3, 00, 00)
+enddate_str = datetime(2023,1,15,3,00,00)
```

Comment:
This is not the string. This is the datetime object, change variable name `startdate_str` to `start_datetime_obj` similar for end...